pkgs/development/tools/parsing/javacc/default.nix

```diff
@@ -21,8 +21,16 @@ stdenv.mkDerivation rec {
     mkdir -p $out/target
     mv scripts $out/bin
     mv target/javacc.jar $out/target/
+    find -L "$out/bin" -type f -executable -print0 \
+      | while IFS= read -r -d ''' file; do
```

Comment:
This would also wrap the bat files which we should probably just rm. Also does this cause double wrapping?

Reply:
The bat file's are not marked executable after the `buildPhase`, that's why I added the `-executable` predicate in find.

Confirmed that there's no double wrapping.

```
$ ls -la /nix/store/z3qfl19qqizzgfay7vk18yb42mr3wb3y-javacc-7.0.12/bin/
total 44
dr-xr-xr-x 2 root root  225 Dec 31  1969 .
dr-xr-xr-x 4 root root   43 Dec 31  1969 ..
-r-xr-xr-x 1 root root  435 Dec 31  1969 javacc
-r--r--r-- 1 root root   94 Dec 31  1969 javacc.bat
-r-xr-xr-x 1 root root  220 Dec 31  1969 .javacc-wrapped
-r-xr-xr-x 1 root root  434 Dec 31  1969 jjdoc
-r--r--r-- 1 root root   93 Dec 31  1969 jjdoc.bat
-r-xr-xr-x 1 root root  219 Dec 31  1969 .jjdoc-wrapped
-r--r--r-- 1 root root 4466 Dec 31  1969 jjrun
-r-xr-xr-x 1 root root  435 Dec 31  1969 jjtree
-r--r--r-- 1 root root   94 Dec 31  1969 jjtree.bat
-r-xr-xr-x 1 root root  220 Dec 31  1969 .jjtree-wrapped
```